packages/ocular-telemetry/src/telemeter.ts

```diff
@@ -114,33 +112,33 @@ class Telemeter {
     return osInfo;
```

Comment:
I think this should be this.osInfo

Reply:
this.osInfo and osInfo are the same here since we are assigning `this.osInfo = osInfo` in the line above

Reply:
I think you forgot to assign `this.osInfo = osInfo` in constructor  , there is  no initialization